The CI jobs are failing because we added a new fuzz target, but that fuzz target (because it's new) doesn't have a corpus .zip file yet:
```
     0K . ............ ..... ................ .......... .make[1]: *** [Makefile:251: corpora/decompress_cross_format_seed_corpus.zip] Error 8
```
Solution is to merge the PR, let OSS fuzz build the corpus, then jobs will be green again. This is annoying, but we add new targets so infrequently that it's not a big deal.

--- a/doc/decompressor_errata.md
+++ b/doc/decompressor_errata.md
@@ -131,7 +131,7 @@ Magicless format
 
 **Last affected version**: v1.5.5
 
-**Affected decompressor component(s)**: Library & CLI
+**Affected decompressor component(s)**: Library
 
 **Produced by the reference compressor**: Yes (example: https://gist.github.com/embg/9940726094f4cf2cef162cffe9319232)
 

--- a/tests/fuzz/decompress_cross_format.c
+++ b/tests/fuzz/decompress_cross_format.c
@@ -77,9 +77,11 @@ int LLVMFuzzerTestOneInput(const uint8_t *src, size_t size)
         // If both accept, decompressed size and data should match
         if (!ZSTD_isError(standardRet) && !ZSTD_isError(magiclessRet)) {
             FUZZ_ASSERT(standardRet == magiclessRet);
-            FUZZ_ASSERT(
-                memcmp(standardDst, magiclessDst, standardRet) == 0
-            );
+            if (standardRet > 0) {
+                FUZZ_ASSERT(
+                    memcmp(standardDst, magiclessDst, standardRet) == 0
+                );
+            }
         }
     }
 
@@ -108,9 +110,11 @@ int LLVMFuzzerTestOneInput(const uint8_t *src, size_t size)
         // If both accept, decompressed size and data should match
         if (standardRet == 0 && magiclessRet == 0) {
             FUZZ_ASSERT(standardOut.pos == magiclessOut.pos);
-            FUZZ_ASSERT(
-                memcmp(standardOut.dst, magiclessOut.dst, standardOut.pos) == 0
-            );
+            if (standardOut.pos > 0) {
+                FUZZ_ASSERT(
+                    memcmp(standardOut.dst, magiclessOut.dst, standardOut.pos) == 0
+                );
+            }
         }
     }
 

--- a/tests/fuzz/decompress_cross_format.c
+++ b/tests/fuzz/decompress_cross_format.c
@@ -57,10 +57,12 @@ int LLVMFuzzerTestOneInput(const uint8_t *src, size_t size)
 
     // Test one-shot decompression
     {
+        FUZZ_ZASSERT(ZSTD_DCtx_reset(dctx, ZSTD_reset_session_and_parameters));
         FUZZ_ZASSERT(ZSTD_DCtx_setParameter(dctx, ZSTD_d_format, ZSTD_f_zstd1));
         const size_t standardRet = ZSTD_decompressDCtx(
                                         dctx, standardDst, dstSize, standardSrc, standardSize);
 
+        FUZZ_ZASSERT(ZSTD_DCtx_reset(dctx, ZSTD_reset_session_and_parameters));
         FUZZ_ZASSERT(ZSTD_DCtx_setParameter(dctx, ZSTD_d_format, ZSTD_f_zstd1_magicless));
         const size_t magiclessRet = ZSTD_decompressDCtx(
                                         dctx, magiclessDst, dstSize, magiclessSrc, magiclessSize);
@@ -88,13 +90,13 @@ int LLVMFuzzerTestOneInput(const uint8_t *src, size_t size)
         ZSTD_outBuffer standardOut = { standardDst, dstSize, 0 };
         ZSTD_outBuffer magiclessOut = { magiclessDst, dstSize, 0 };
 
+        FUZZ_ZASSERT(ZSTD_DCtx_reset(dctx, ZSTD_reset_session_and_parameters));
         FUZZ_ZASSERT(ZSTD_DCtx_setParameter(dctx, ZSTD_d_format, ZSTD_f_zstd1));
         const size_t standardRet = ZSTD_decompressStream(dctx, &standardOut, &standardIn);
-        ZSTD_DCtx_reset(dctx, ZSTD_reset_session_only);
 
+        FUZZ_ZASSERT(ZSTD_DCtx_reset(dctx, ZSTD_reset_session_and_parameters));
         FUZZ_ZASSERT(ZSTD_DCtx_setParameter(dctx, ZSTD_d_format, ZSTD_f_zstd1_magicless));
         const size_t magiclessRet = ZSTD_decompressStream(dctx, &magiclessOut, &magiclessIn);
-        ZSTD_DCtx_reset(dctx, ZSTD_reset_session_only);
 
         // Standard accepts => magicless should accept
         if (standardRet == 0) FUZZ_ASSERT(magiclessRet == 0);

